modules/ROOT/pages/provisioning-virtualbox.adoc

```diff
@@ -0,0 +1,191 @@
+= Provisioning Fedora CoreOS on VirtualBox
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on the VirtualBox hypervisor.
```

Comment:
It feels like we should right upfront mention that FCOS currently does not ship VirtualBox images but may in the future and that what follows are platform-agnostic ways to set up Fedora CoreOS as a bare metal system.

Reply:
That's a good point. Would it be alright to add that hint together with links to the two issues ([#144](https://github.com/coreos/fedora-coreos-tracker/issues/144), [#73](https://github.com/coreos/fedora-coreos-tracker/issues/73)) related to providing VirtualBox images?

Reply:
Yes, that sounds great!

Reply:
I updated the pull request with such a hint.